readme.MD

```diff
@@ -18,3 +18,8 @@ Default bases include (landing-page, portfolio-page)
 Components: components are children of Bases. A Component must be a child of at least one Base.
 Components are invoked via --flags.
 Default components include (navbar, navbar-searchbar, sidebar, footer, footer-big, --background-video)
+
+
+Contributors: 
```

Comment:
This should be a header no? Like so:
# Contributors:

Reply:
This is true